java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/PrebuiltRuntimeImageBuildStep.java

```diff
@@ -54,7 +63,31 @@ protected String getArtifact(BuildContext buildContext) throws BuildStepExceptio
     } else if (artifacts.size() > 1) {
       throw new TooManyArtifactsException(artifacts);
     } else {
-      return buildContext.getWorkspaceDir().relativize(artifacts.get(0)).toString();
+      Path artifact = artifacts.get(0);
+      logger.debug("Found Java artifact {}", artifact);
```

Comment:
:+1: Yep, this is because `getArtifact` is being called twice. I'll make sure the log message only appears once.